x-pack/plugins/secops/server/lib/events/elasticsearch_adapter.ts

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get, has, isEmpty, merge } from 'lodash/fp';
+import { EventItem, EventsData, KpiItem } from '../../../common/graphql/types';
+import { FrameworkAdapter, FrameworkRequest } from '../framework';
+import {
+  EventData,
+  EventFilterQuery,
+  EventsAdapter,
+  EventsRequestOptions,
+  TermAggregation,
+} from './types';
+
+const EventFieldsMap = {
+  timestamp: '@timestamp',
+  'host.hostName': 'host.name',
```

Comment:
What are the thoughts around why this rename is necessary?

Reply:
Trying to match ecs Host fields https://github.com/elastic/ecs#host

I thought that what we wanted to do and since all our data in ES does not match ECS stuff I need to do some mapping

Reply:
I understand now: the `ECS` version of the field is not in prod yet, so we're mapping to the prod values for now.

Reply:
let's make `hostName` match the `ECS` `hostname`

Reply:
👍